### ISSUE TITLE
[BUG] Confirmation/Dismissal Announcements

### ISSUE THREAD

**Issue by themattyg** [fymmot/inclusive-dates#84]
**Describe the bug**
Dismissal of the date range modal does not announce confirmation with screen readers.

**Reproduction**
Selection (with the ENTER key) or dismissal (with the ESC key) on date range is announced with a screen reader (such as NVDA). When confirming a date by adjusting from the date range field itself (using the ENTER key) there is no screen reader confirmation. The same goes for when the date range modal is dismissed using the ESC key—no announcement.

**Expected behavior**
The expectation is that both opening and closing of the modal is confirmed by announcement.

**Context**
W3C ARIA APG references aria-label=”string” (setting an initial “choose date” value and changing it to the “date_string” when selected) but we’re not seeing where the hole is with the confirmation and dismissal not getting picked up by screen readers. (Everything else is announced flawlessly, including date selection.)

- Device: Desktop PC
- Operating System: Windows 10
- Browser: Chrome
- Example implementation: https://uhn.dev.sandboxsoftware.ca/events/ (Date Range Field)
